saleor/graphql/checkout/subscriptions.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
```

Comment:
Maybe worth to have 1:1 stucture representation that we have for webhook?
So we have `checkout/webhooks/list_shipping_methods.py`, then here we could have `graphql/checkout/subscriptions/list_shipping_methods.py` ?